### ISSUE TITLE
Add more and better options for displaying images on a docs page

### ISSUE THREAD

**Issue by paperclypse** [newrelic/docs-website#4591]
## Is your feature request related to a problem? Please describe

Currently, images display at their actual size, center-aligned and that's it.

Austin: There's little ability to flow images inline with the page.

## Describe the solution you'd like

I'd love to have a set of style options for images:
* Resize/scale
* Alignment (left, center, right)
* More readable captions (c.f. #4658)
* Text wrap on/off

Also, it would be great to be able to handle SVG images more elegantly, because they scale more elegantly.

## Describe why this important to you

I don't think the images on our site look that great in how they display on the page. The sizing is inconsistent and our site doesn't handle large or small images very well. I'd love for them to look better. 

When there are a lot images, it can seem kind of a jumble: https://docs.newrelic.com/docs/codestream/how-use-codestream/start-work/

**Comment by rhetoric101** [comment]
I would like to echo the request of @paperclypse. I would prefer to have the option to left-justify images within the numbered lists (procedures). In other words, could we have the option to keep images within the bounds of the numbered list items but justified to the left so they match the  indentation of the number?

**Comment by rhetoric101** [comment]
I also had a recent experience where I needed to center images in a table column so they would like up correctly. To accomplish, I inserted inline table formatting, which isn't a sustainable approach. Instead, we need some way to specify a class that enables us to center images at will. Here's that topic where I did that: https://docs.newrelic.com/docs/new-relic-one/use-new-relic-one/ui-data/automaps/#open-automap
